bundles/ch.elexis.core.ui.contacts/src/ch/elexis/core/ui/contacts/views/KontaktBlatt.java

```diff
@@ -273,12 +283,103 @@ public void linkActivated(HyperlinkEvent e) {
 		lbAnschrift = tk.createLabel(cAnschrift, StringConstants.EMPTY, SWT.WRAP);
 		lbAnschrift.setLayoutData(SWTHelper.getFillGridData(1, true, 1, false));
 		setOrganisationFieldsVisible(false);
+		def[14].getWidget().setVisible(false); // field is only added for UI presentation reasons
 		def[19].getWidget().setVisible(false); // field is only added for UI presentation reasons
 		setUnlocked(false);
 
 		CoreUiUtil.injectServicesWithContext(this);
 	}
 
+	/**
+	 * Updates the view of the fields within the {@code afDetails} form:
+	 * <p>
+	 * <ul>
+	 * <li>It adjusts the width of the {@link ColumnLayoutData} for each
+	 * {@link LabeledInputField} based on the length of its label.</li>
+	 * </ul>
+	 * After these updates, the method refreshes the tooltip text for each field and
+	 * lays out the controls in the container.
+	 * </p>
+	 * 
+	 * @author mdedic
+	 * @since 3.13
+	 */
+	private void updateFieldsView() {
+		if (afDetails != null && !afDetails.isDisposed()) {
+			Control[] controls = afDetails.getChildren();
+			for (Control control : controls) {
+				if (control instanceof LabeledInputField) {
+					LabeledInputField field = (LabeledInputField) control;
+					ColumnLayoutData data = (ColumnLayoutData) field.getLayoutData();
+					if (data == null) {
+						data = new ColumnLayoutData();
+					}
+					int width = field.getLabelComponent().getText().length() * 7;
```

Comment:
Hier fix 7 als Breite eines Zeichnes des Font zu nehmen ist nicht ideal.

Es gibt die Methode `stringExtent` in der Klasse `GC`, um die Breite eine String zu erhalten. Bitte diese verwenden. Ideal wäre eine Methode `CoreUIUtil.getStringExtent(Drawable d, String text)`, und darin wird ein `GC` erzeugt um die Breite zu ermitteln, und danach mit `GC.dispose` wieder aufgeräumt. Dann könnte das auch an anderen Stellen wieder verwendet werden.